src/components/Book/Book.module.css

```diff
@@ -2,14 +2,177 @@
+  background-color: #fff;
+}
+
+.AboutBook {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.BookInfo {
+  display: flex;
+  flex-direction: column;
+  gap: 0.25rem;
+}
+
+.Category {
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.BookInfoWith {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.BookOperation {
+  list-style: none;
+  display: flex;
+  gap: 0.25rem;
+}
+
+.Operate {
+  border: 2px solid transparent;
+}
+
+.Author,
+.Operation {
+  display: block;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background-color: transparent;
+  border: none;
+}
+
+.Operate:nth-child(even) {
+  padding-left: 0.45rem;
+  padding-right: 0.45rem;
+  border-left-color: #1212124a;
+  border-right-color: #1212124a;
+}
+
+.ChapterStatus {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.Status {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+}
+
+.Chapter {
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  text-transform: uppercase;
+}
+
+.ChapterNumber {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.ProgressUp {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  border: none;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+}
+
+.ProgressCircle {
+  display: flex;
+  gap: 1rem;
+  padding: 0 4rem;
+  border: 4px solid transparent;
+  border-right-color: rgba(0, 0, 0, 0.055);
+}
+
+.Circle {
+  width: 4.25rem;
+  height: 4.25rem;
+  border-radius: 50%;
+  border-style: solid;
+  border-width: 5px;
+  border-color: #12121246;
+}
+
+.Rate {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Mark {
+  display: block;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
 }
```

Comment:
[OPTIONAL] One of the Bad CSS Practices is using the Same Rule Repetitively. I noticed that you use repetitive rules for each element. If you want to create a rule for multiple elements, use a class attribute instead.
For instance, instead of repeating font params (font-weight, font-style...) for each class:
```css
.Rate {
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  letter-spacing: normal;
}

.Mark {
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  letter-spacing: normal;
}
```
Use a class instead. Ex:
```css
.font-params {
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  letter-spacing: normal;
}
```